go-controller/pkg/config/config.go

```diff
@@ -188,6 +195,16 @@ type LoggingConfig struct {
 	ACLLoggingRateLimit int `gcfg:"acl-logging-rate-limit"`
 }
 
+// MonitoringConfig holds monitoring-related parsed config file parameters and command-line overrides
+type MonitoringConfig struct {
```

Comment:
@rcarrillocruz you don't need to do it now, but I noticed you're parsing the flow targets twice. The way pkg/config handles that for other options is to define a RawXXXX struct member that has the `gcfg:xxx` bits, and then an XXXX parsed/validated member that the Go code actually uses. The the config reading code validates, parses, and sets the non-Raw member. See HybridOverlayConfig and ClusterSubnets for an example of that.

Reply:
Done